GATK_QuickByte.md

```diff
@@ -320,35 +361,18 @@ Here is a sample PBS script combining everything we have above, with as much par
 
 	cat $src/sample_list | env_parallel -j $PBS_NUM_NODES --sshloginfile $PBS_NODEFILE \
 		'bwa mem \
-			-t $threads -M \
-			-R "@RG\tID:{}\tPL:ILLUMINA\tLB:{}\tSML{}" \
+			-t 8 -M \
+			-R "@RG\tID:{}\tPL:ILLUMINA\tLB:{}\tSM:{}" \
 			$reference \
 			$src/raw_reads/{}_R1.fastq.gz \
```

Comment:
Your script uses the trimmed reads. Did you mean to keep the raw reads here?

Reply:
Meant to be clean, fixed!